lr-cli/cortx_setup/commands/storage/commons.py

```diff
@@ -56,3 +59,41 @@ def get_enc_id(self, targets=None, force=False):
                 return self.fetch_enc_id(targets)
             except:
                 raise exception("Error generating the enclosure ID")
+
+
+class EnclosureInfo(Command):
```

Comment:
Move this class to a separate file. It improves the readability.

Reply:
Done